src/Microphysics2M.jl

```diff
@@ -421,7 +421,7 @@ Fall velocity of individual rain drops is parameterized:
 """
 function rain_terminal_velocity(
     (; pdf_r)::CMP.SB2006{FT},
-    (; ρ0, aR, bR, cR)::CMP.SB2006VelType{FT},
+    (; ρ0, aR, bR, cR, modified)::CMP.SB2006VelType{FT},
```

Comment:
I'm wondering, does it even make sense to keep the `modified=false` as option? It seems to me like we should always be computing it this way. - The code would be simpler and without 4 `if/else` statements there. 

If not, it would be better to define two different versions of this function and dispatch based on the velocity type. This way the choice would happen at the compile time and not via `if/else` during run time.

Reply:
The reason I defined this flag is because I thought maybe Caterina is using the original SB2006. So I kept the original as a default. But if not I'm also happy to change the function to the new form.
@crocicc Do you want to keep the original sedimentation equation?